scope3_methodology/publisher/model.py

```diff
@@ -83,9 +83,6 @@ class Property(CustomBaseModel):
     revenue_allocation_to_digital_pct: Optional[Decimal] = field(
         default=None, metadata={"default_eligible": True}
     )
-    revenue_allocation_to_display_pct: Optional[Decimal] = field(
-        default=None, metadata={"default_eligible": True}
-    )
     revenue_allocation_to_ads_pct: Optional[Decimal] = field(
         default=None, metadata={"default_eligible": True}
     )
```

Comment:
`revenue_allocation_to_display_pct` was not being used at all? 

Do we need both `revenue_allocation_to_ads_pct` and `revenue_allocation_to_digital_pct` ?